src/main/resources/db/migration/V202409231158__create_table_feedback.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE feedback (
+    id SERIAL PRIMARY KEY,
+    employee_id INT NOT NULL,
+    company_id INT NOT NULL,
+    date_of_feedback DATE DEFAULT current_date,
+    anonymity VARCHAR(255) NOT NULL,
+    reviewed BOOLEAN NOT NULL DEFAULT false,
+    feedback VARCHAR(255) NOT NULL,
+    response_id INT[]
```

Comment:
what is this column?

Reply:
each feedback has a list of all responses of that feedback. So it holds a list of response id's

Reply:
and you also hold the feedback_id in the response?
why do you need both?

Reply:
It's a good point. I thought that way each feedback is linked directly to all it's responses, but it's not necessary if i have the feedback id in response